maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -186,6 +190,13 @@ private DistroProperties resolveDistroProperties(Server server) throws MojoExecu
 					}
 					platformMode = false;
 					break;
+				case O3_REFERENCE_APPLICATION:
+					distroProperties = DistroHelper.getDistroPropertiesFromFile(new File(SDKConstants.MAVEN_PLUGINS_RESOURCE_PATH + "openmrs-distro-3.0.properties"));
```

Comment:
We don't want to embed a specific distro.properties in the distribution. It would be nice to be able to publish a distribution for O3 in the same way we do for O2, e.g., via the referenceapplication-package artifact having an `openmrs-distro.properties` file. But, we need support for downloading and expanding the Iniz config first.